.lando.yml

```diff
@@ -7,48 +7,49 @@ config:
   php: '8.1'
   conf:
     php: lando/php.ini
-
 services:
   appserver:
     xdebug: true
     overrides:
       environment:
-        # Support debugging with XDEBUG 3.
         XDEBUG_MODE: debug
-        # Support debugging Drush with XDEBUG 3.
-        PHP_IDE_CONFIG: "serverName=xsmind.lndo.site"
-        # Specific Xdebug Configs
-        LANDO_HOST_IP: "host.docker.internal"
-        XDEBUG_CONFIG: "remote_enable=1 remote_host=host.docker.internal"
+        PHP_IDE_CONFIG: serverName=xsmind.lndo.site
+        LANDO_HOST_IP: host.docker.internal
+        XDEBUG_CONFIG: remote_enable=1 remote_host=host.docker.internal
     config:
       php: lando/php.ini
   phpmyadmin:
     type: phpmyadmin
   database:
     run_as_root:
-      # Assign permission databases.
-      - mysql -uroot -e "GRANT SUPER ON *.* to 'drupal9'@'%' IDENTIFIED by 'drupal9';"
-
+      - >-
```

Comment:
Line number 25 . what is this. what this symbol do ?

Reply:
@archanarkamath  whenI took pull from your branch to my local. I found that there are many garabge things you have pushed. If it would have only single file then I could have fix it and merge.

Check lando file line 25,43 & 52.
not only this check menu--acount.html.twig. line 71.

go to xsmind application. clicked on Logout. then you will get pop up to cofirm user. are you sure ? want to logout.
check that popup what is happening there with UI. because of this garbage code.

If I merge this PR lots of rework. Hence rejecting this PR. Please create a new branch and work in clean way.